images/linux/scripts/tests/Tools.Tests.ps1

```diff
@@ -119,3 +119,13 @@ Describe "gfortran" {
         "$GfortranVersion --version" | Should -ReturnZeroExitCode
     }
 }
+
+Describe "codeql" {
+    It "codeql" {
+        $codeql_bundle_name = "$(curl -sSL https://raw.githubusercontent.com/github/codeql-action/main/src/defaults.json | jq -r .bundleVersion)"
```

Comment:
Why do we need to query version if we can invoke `codeql version` command?

Reply:
Agree may be we can even unify this validation with other toolset tools?

Reply:
codeql  is not added to path so I guess that we have to get actual binary location

Reply:
@LeonidLapshin we don't need it in PATH. You should just check all versions that live in `$env:AGENT_TOOLSDIRECTORY/CodeQL`

Reply:
See https://github.com/actions/virtual-environments/blob/main/images/win/scripts/Tests/Toolset.Tests.ps1
I think it is worth to unify with other toolset tools

Reply:
thank you, I'll check it

Reply:
moved CodeQL tests to toolset